ProjFS.Mac/PrjFSKext/public/PrjFSPerfCounter.h

```diff
@@ -36,6 +36,8 @@ enum PrjFSPerfCounter : int32_t
                 PrjFSPerfCounter_FileOp_FindRoot,
                     PrjFSPerfCounter_FileOp_FindRoot_Iteration,
                 PrjFSPerfCounter_FileOp_ShouldHandle_NoRootFound,
+            PrjFSPerfCounter_FileOp_ShouldHandle_CheckRootOnline,
+                PrjFSPerfCounter_FileOp_ShouldHandle_OfflineRoot,
```

Comment:
The `PerfCounterNames` array in kext-perf-traciing.cpp needs to be updated to include these new values as well.

Reply:
Thanks for catching that, I'll fix that up shortly.

Reply:
Fixed. And so that I can't make that mistake again, posted #887.